core/src/main/java/org/geysermc/geyser/util/EntityUtils.java

```diff
@@ -153,6 +154,9 @@ private static float getHeightOffset(Entity passenger) {
      * Adjust an entity's height if they have mounted/dismounted an entity.
      */
     public static void updateMountOffset(Entity passenger, Entity mount, boolean rider, boolean riding, boolean moreThanOneEntity) {
+        if(passenger instanceof TextDisplayEntity textDisplay
+                && textDisplay.baseTranslation == null ) return;
+
```

Comment:
As per our contributer note, please follow the formatting guidelines: 
https://github.com/GeyserMC/Geyser/blob/master/CONTRIBUTING.md

```suggestion
        if (passenger instanceof TextDisplayEntity textDisplay
                && textDisplay.baseTranslation == null) return;

```